apps/hashdotdev/src/_pages/blog/0012_block-protocol-0-3-wordpress.mdx

```diff
@@ -0,0 +1,65 @@
+
+## How does the Block Protocol in WordPress actually work?
+
+The Block Protocol for WordPress plugin works by effectively turning WordPress into an embedding application for the Block Protocol. It introduces no breaking changes to WordPress sites, and you can continue to update WordPress as normal.
+
+The plugin sets up a table in the WordPress database of any site using it. In this table, **entities** and **links** are stored. API routes are then added that allow Block Protocol blocks to use that table.
+
+Block Protocol blocks are dynamically fetched from the [Þ Hub](https://blockprotocol.org/wordpress) and registered in WordPress, appearing like native blocks in WordPress’s normal block editor (known as “Gutenberg”).
+
+When a Block Protocol block is selected, the plugin loads that block and stores its data in the new `entity` table in the database. With that data stored, the block will load in edit mode when opened from post/page editor, and in read-only mode when viewed on the live website, or in preview.
+
+WordPress admins can choose to allow users to insert any Block Protocol block on their site, or limit the selection to pre-approved (e.g. security-vetted) blocks only.
+
+## What WordPress support means
+
+**For users:** Making the entire catalogue of Þ blocks available in WordPress means that, in just a few clicks, blocks developed for the Block Protocol can be used by the 43% of websites on the internet powered by WordPress.
+
+**For developers:** The WordPress plugin also demonstrates the interoperability of Block Protocol blocks. Existing and new blocks on the [Þ Hub](https://blockprotocol.org/wordpress) now work in both [HASH](https://blockprotocol.org/docs/using-blocks#hash) and WordPress, without any application-specific logic, and next-up we’ll be working on both [Figma & FigJam](https://blockprotocol.org/docs/using-blocks#figma)V plugins, as well as integration with [GitHub Blocks](https://blockprotocol.org/docs/using-blocks#github-blocks), providing even more environments for Þ blocks to be used in. If you’re thinking about developing a new block for WordPress, we’d strongly encourage you to consider [building a Þ block](https://blockprotocol.org/docs/developing-blocks) instead, allowing your block to be used not only within WordPress, but all of these environments (as well as future ones TBC).
+
+## External services & APIs in blocks
+
+The _AI Text_ and _AI Image_ blocks are powered by OpenAI GPT-3 Davinci, and OpenAI DALL-E respectively.
+
+- The AI text block lets you to quickly generate content outlines, as well as actual prose, and can assist in rewriting chunks of existing text to match a particular style or level of quality.
```

Comment:
> can assist in rewriting chunks of existing text to match a particular style or level of quality

Not at the moment, although a good future feature

Reply:
I mean, it can if you copy and paste manually in and provide a prompt "rewrite this in style X". I'm going to leave this language.